### DISCUSSION TITLE
训练OCRv4_det_cml，报错resize() got multiple values for argument 'interpolation'

### DISCUSSION THREAD

**Discussion by yanshuikafei** [PaddlePaddle/PaddleOCR#14289] [Q&A]
训练OCRv4_det_cml，报错resize() got multiple values for argument 'interpolation'
![Uploading 图片.png…]()

**Comment by yanshuikafei** [comment]
![046A93F4-7BC6-4299-964F-2A14E559B35A](https://github.com/user-attachments/assets/4193da37-8e39-4274-ad43-80f0c5e50bfa)

**Reply by GreatV** [reply]
配置文件有改动吗，可以换成paddleocr2.9.1试试

**Reply by yanshuikafei** [reply]
谢谢，那我一会换成2.9.1试试；
没有任何改动，数据集也是从咱们飞桨平台上下载的。

**Reply by 759325100** [reply]
@yanshuikafei @GreatV 
请问这个问题你有解决吗，我是用最新的官方代码，也安装了最新的paddlepaddle，这个错误依然存在

**Reply by 759325100** [reply]
我是用的训练配置文件为：configs/deb/ch_PP-OCRv4/ch_PP-OCRv4_det_student.yml

**Reply by 759325100** [reply]
与训练模型使用的是ch_PP-OCRv4_det_server_train/best_accuracy.pdparams

**Reply by GreatV** [reply]
你可以换成2.9.1

**Reply by GreatV** [reply]
这个是main分支的问题

**Comment by 759325100** [comment]
希望得到大佬们的答复，感谢

**Comment by GreatV** [comment]
无法复现
![image](https://github.com/user-attachments/assets/b5fe4547-4cc5-4737-b554-1d5baa4d8ffd)
可以检查一下
```
albumentations==1.4.10
# to be compatible with albumentations
albucore==0.0.13
```
